src/System.Private.CoreLib/src/System/Threading/RuntimeThread.cs

```diff
@@ -2,33 +2,123 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
+using System.Diagnostics;
+using System.Runtime;
+using Microsoft.Win32.SafeHandles;
 
 namespace System.Threading
```

Comment:
CoreFX Thread wraps it, it does not derive from it.

https://github.com/dotnet/corefx/blob/277b781b6326e06716173132f68317ddc475f67d/src/System.Threading.Thread/src/System/Threading/Thread.cs#L32